src/material/list/list.ts

```diff
@@ -160,6 +167,8 @@ export class MatListSubheaderCssMatStyler {}
   exportAs: 'matListItem',
   host: {
     'class': 'mat-list-item',
+    '[class.mat-list-item-disabled]': 'disabled',
+    '[attr.aria-disabled]': 'disabled',
```

Comment:
Not completely sure whether we want this. I added it because the list items can also be links.

Reply:
It would be invalid to put `aria-disabled` on something that doesn't have a role (implicit or explicit), so we should probably want to at least avoid applying an invalid state